src/Lexer.js

```diff
@@ -75,14 +75,14 @@ const tokenRegex = new RegExp(
     "([ \r\n\t]+)|" +                                 // whitespace
     "([!-\\[\\]-\u2027\u202A-\uD7FF\uF900-\uFFFF]" +  // single codepoint
     "|[\uD800-\uDBFF][\uDC00-\uDFFF]" +               // surrogate pair
-    "|\\\\(?:[a-zA-Z]+|[^\uD800-\uDFFF])" +           // function name
+    "|\\\\(?:[a-zA-Z@]+|[^\uD800-\uDFFF])" +          // function name
```

Comment:
FWIW, The MathJax [mhchem extension](https://github.com/mhchem/MathJax-mhchem/blob/master/mhchem.js) makes heavy use of `@`. I'm no expert, so I can't tell you if it is being used as a catcode.